src/components/Routes.tsx

```diff
@@ -244,6 +251,27 @@ export function Routes() {
         path="/admin/unrelease"
         render={(routeProps) => <Unrelease key={routeProps.location.key} />}
       />
+      <AuthenticatedRoute
+        path={"/lab/fetal_waste"}
+        render={(routeProps) => (
+          <RecordInPlace
+            title={"Fetal Waste"}
+            operationType={"Convert to fetal waste"}
+            key={routeProps.location.key}
+            columns={[
+              columns.barcode(),
+              columns.labwareType(),
+              columns.tissueType(),
+              columns.spatialLocation(),
+              columns.replicate(),
+            ]}
+            description={
+              " to change bio state of all scanned labware to 'Fetal Waste" +
+              "                        Waste'"
```

Comment:
Looks like the word Waste is repeated here...?

Reply:
Yes, corrected. Thank you